core/src/commonMain/kotlin/com/ustadmobile/core/controller/ContentEntryList2Presenter.kt

```diff
@@ -69,13 +83,13 @@ class ContentEntryList2Presenter(context: Any, arguments: Map<String, String>, v
                 Role.PERMISSION_CONTENT_UPDATE)
     }
 
-    private fun getAndSetList(sortOrder: SortOrder = currentSortOrder) {
+    private fun getAndSetList(sortOrder: SortOrder = currentSortOrder, showHidden: Boolean = false) {
```

Comment:
Why is showHidden a function parameter, and the others are member variables? Let's use the same pattern for all: e.g. showHidden should be a member variable like parentEntryUid, loggedinPersonUid, onlyFolderFilter, etc.